rest-api-spec/src/main/resources/rest-api-spec/api/security.disable_user_profile.json

```
@@ -0,0 +1,41 @@
+{
+  "security.disable_user_profile":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-disable-user-profile.html",
```

This URL won't exist on the `current` docs branch until 8.2 is released. We'll need to use the main/master branch in the meantime. This'll ensure our client docs don't break.
```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-disable-user-profile.html",
```

---

This is good to know. But do we need to change it back to `current` once 8.2 is released?

---

You can leave it in. We have several other older links pointing to `/master/` I'll make a note for the docs team to clean up these links after the 8.2 release.